execution-env.md

```diff
@@ -469,6 +469,15 @@ The following sections describe differences from specific SPIR-V instructions.
 *   **OpSpecConstantOp**:
   * Opcode **OpVectorShuffle** may not have a component literal with value 0xFFFFFFFF.
 
+####  Memory instructions
+
+* Workgroup variables are only usable in **GLCompute** shaders:
```

Comment:
instead of using a ':', should it be ', additionally' or ', specifically'

The transition here feels awkward to read, since I am unclear if you are stating a formal definition of what the first line is saying, or specifying an additional restriction.